api/src/main/java/io/strimzi/api/kafka/model/rebalance/KafkaRebalanceStatus.java

```diff
@@ -30,12 +31,12 @@
 @ToString(callSuper = true)
 public class KafkaRebalanceStatus extends Status {
     private String sessionId;
-    private Map<String, Object> optimizationResult = new HashMap<>(0);
+    private Map<String, Object> optimizationResult;
 
     @Description("A JSON object describing the optimization result")
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     public Map<String, Object> getOptimizationResult() {
-        return optimizationResult;
+        return optimizationResult != null ? optimizationResult : emptyMap();
```

Comment:
Map.of()? Or does it need to be mutable?

Reply:
I missed that one, thanks. Looks like KafkaRebalanceAssemblyOperatorTest likes both emptyMap() and Map.of(), which are immutable.